config/prod/nf-syn28545963-s3.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.8/templates/S3/synapse-external-bucket.j2"
+stack_name: "nf-syn28545963-s3"
+stack_tags:
+  Department: "SCCE"
+  Project: "neurofibromatosis"
+  OwnerEmail: 'robert.allaway@sagebionetworks.org'
+  CostCenter: "CTF NF Amend 11 / 304002"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/robert.allaway@sagebase.org'
+    - 'arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/sasha.scott@sagebase.org'
```

Comment:
we are planning deprecate sandbox and use identity-central account for cross account access instead.  we are just waiting on PR https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/603 to merge.

Reply:
Thanks Khai!

Reply:
@allaway here are the ARNs you'll need..

```
arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/robert.allaway@sagebase.org

arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/sasha.scott@sagebase.org

```

Reply:
```suggestion
    - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/robert.allaway@sagebase.org'
    - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/sasha.scott@sagebase.org'
```

Reply:
Thanks Khai (again)!